### PR TITLE
Material UI - Program Tags

### PR DESCRIPTION
**Pull request by cabarnes**
Convert program tags to Material UI

image

### CODE REVIEW

File: src/components/ProgramTags.js [hunk 1]
```diff
@@ -54,72 +59,46 @@ class ProgramTags extends Component {
...
     return (
-      <Grid centered container>
-        <Grid.Row>
-          <Dropdown
-            fluid
-            search
-            selection
+      <Grid container direction="column" justify="center" alignItems="stretch" spacing={1}>
+        <Grid item>
+          <Autocomplete
```
Comment (aninternetof) [inline src/components/ProgramTags.js:75]: This works well.

**Comment by aninternetof** [comment]
It may not be the right time to make this change, but would it very tricky to completely remove the save button and have it auto-save after I pick a tag from the list?

**Comment by aninternetof** [comment]
It looks like there's not a way for me to add a new tag anymore. *But*, I think I like that. Us admins can use the console to add tags, and regular users can just select from the existing ones for now. Otherwise our dropdown would get way to cluttered.

**Comment by cabarnes** [comment]
It should allow any text to be added as a tag like before

**Comment by aninternetof** [comment]
image
If I type something not from the list, I'm not seeing a way to save the tag.

**Comment by cabarnes** [comment]
You should just be able to hit Enter

**Comment by aninternetof** [comment]
Oh, duh. That works.

**Comment by cabarnes** [comment]
So, should it be removed?

**Comment by aninternetof** [comment]
Nope, let's leave it. We'll see if it gets cluttered early on.
And really, the autocomplete makes it manageable even if it gets cluttered.

**Comment by cabarnes** [comment]
Should be simple enough to save on the change event and remove the button. There will be more API calls in total but it may not be something lots of users are changing

**Pull request merged**